pkg/pillar/hypervisor/kvm.go

```diff
@@ -300,8 +303,8 @@ func newKvm() Hypervisor {
 		return kvmContext{
 			domains:      map[string]int{},
 			devicemodel:  "pc-q35-3.1",
-			dmExec:       "qemu-system-x86_64",
-			dmArgs:       []string{"-display", "none", "-daemonize", "-S", "-no-user-config", "-nodefaults", "-no-shutdown", "-no-hpet"},
+			dmExec:       "/usr/lib/xen/bin/qemu-system-x86_64",
```

Comment:
Why do you need to add full path here?

Reply:
Since /usr/lib/xen/bin is not a standard $PATH, I had to mention the absolute path for finding qemu-system-x86_64 during task launch. Currently it is working because from pillar we are launching /usr/bin/qemu-system-x86_64, which actually makes the cross-container call.

Reply:
Ah! Ok -- makes sense